systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -65,29 +59,18 @@
             "internal.value.converter.schemas.enable=false\n" +
             "internal.value.converter=org.apache.kafka.connect.json.JsonConverter\n";
 
-    @Test
-    @JUnitGroup(name = "regression")
-    @Resources(value = "../examples/templates/cluster-operator", asAdmin = true)
-    @OpenShiftOnly
-    public void testDeployConnectClusterViaTemplate() {
```

Comment:
Because of duplication with https://github.com/strimzi/strimzi-kafka-operator/blob/6c39e3604512a80a0dea489b3d91f8cd14b8bc81/systemtest/src/test/java/io/strimzi/systemtest/OpenShiftTemplatesST.java#L180